docs/root/intro/arch_overview/upstream/upstream_filters.rst

```diff
@@ -4,8 +4,10 @@ Upstream network filters
 ========================
 
 Upstream clusters provide an ability to inject network level (L3/L4)
-:ref:`filters <arch_overview_network_filters>`. The filters apply to the
-connection to the upstream hosts, using the same API presented by listeners for
+filters. It should be noted that network fitler needs to
```

Comment:
I was going to suggest some spelling and grammar tweaks, but I'm actually not sure about the semantics of this change.

I think I'll defer to @mattklein123 for this.

Reply:
> I was going to suggest some spelling and grammar tweaks
Yeah, I'm not sure what would be the best way to convey the information from #11015 in the docs.

I'm also not sure if the part about "registering upstream filters in the code" is necessary, but it felt strange to just add that there's no upstream filters without an explanation.

Reply:
I think this is OK with me, modulo the spelling fixes needed (fitler). Thank you!

/wait